examples/ios_app/ExampleResources/dep_resources_collector.bzl

```diff
@@ -0,0 +1,49 @@
+"""An example aspect to exercise Xcode target selection."""
+
+# buildifier: disable=bzl-visibility
+load("@build_bazel_rules_apple//apple/internal:resources.bzl", "resources")
```

Comment:
I did this here since a user of these rules was doing it. I think you can manually create an `AppleResourceInfo` instead. I'm not sure this function should become public API. I'll try to not use it here, and if I fail, I'll open an issue on rules_apple.

Reply:
https://github.com/bazelbuild/rules_apple/issues/1412